README.md

```diff
@@ -1,27 +1,35 @@
-#Python-Macros-For-FreeCAD
-This repository contain Python scripts to design Parts on FreeCAD. 
-Instead of using a mouse to create 3D parts, Python scripts can create 3D parts instantly with FreeCAD.
+## Python-Macros-For-FreeCAD
 
-#Requirements:
-1) Visual Studio Code : https://code.visualstudio.com/
-2) FreeCAD : https://www.freecadweb.org/
-3) Python : https://www.python.org/
+This repository contain Python scripts to design Parts on FreeCAD, an opensource computer aided design (CAD) software. Instead of using a mouse to create 3D parts, Python scripts are used to autogenerate 3D parts instantly utilizing FreeCAD.
 
-#How to launch the snippets?
-1) Open FreeCAD.
-2) Select Part with the "Switch between workbenches".
-3) Click on View button.
-4) Click on Panels.
-5) Click on Python Console.
-6) Click on the "Create a new empty document" button.
-6) Type the command-line : exec(open("the_path_of_your_file").read()).
-7) Click on the right button on your mouse.
-8) Click on Fit all.
+## Background
 
-The part was created by the code.
+This repository's aim is to publish free energy devices in CAD to bring free energy devices to the world in order to escape poverty globally, maintain all eco-systems and escape censorship.
 
-All parts can be seen on Holomorphe's website (https://www.holomorphe.com - Computer aided designed) with BabylonJS viewer (https://doc.babylonjs.com/extensions/babylonViewer) with .stl format.
+This repository publishes free energy devices invented by inventors like Stanley Meyer, Nikola Tesla, John Bedini, Viktor Schauberger, Chas Campbell, Edwin Gray, Thomas Henry Moray and others inventors who worked on free energy devices.
 
-This repository wants to publish free energy devices in cad to bring free energy devices to the world in order to escape poverty globally, maintain all eco-systems and escape censorship.
+## Requirements
+
+1. Visual Studio Code: https://code.visualstudio.com/
+1. FreeCAD: https://www.freecad.org/ version 0.19 or higher
+1. Python: https://www.python.org/ version ??
```

Comment:
Minimum version of python is based on the minimum version of FreeCAD you're using